File: C#/2373 Garçom.cs
using System;

class MainClass 
{
  public static void Main (string[] args) 
  {
    int n = int.Parse(Console.ReadLine());
    int copos = 0;
    for(int i = 0; i < n; i++){
        string[] s = Console.ReadLine().Split();
        if(int.Parse(s[0]) > int.Parse(s[1])) copos += int.Parse(s[1]);
    }
    Console.WriteLine(copos);
  }
}

File: C++/2373 Garçom.cpp
#include <iostream>
using namespace std;
int main(){
    int copos = 0;
    int n, a, b;
    cin >> n;
    for(int i = 0; i < n; i++){
        cin >> a >> b;
        if(a > b) copos += b;
    }
    cout << copos << endl;
}